#goal definition
std_msgs/Header header
bool modality # 1: pick-up target_object, 0: place picked-up object
assignment2/PickUpObject target_object
assignment2/PickUpObject[] objects
assignment2/PickUpObstacle[] obstacles
---
#result definition
std_msgs/Header header
bool is_succeeded
---
#feedback
std_msgs/Header header
string status
